sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/ContainerAsyncApiTests.java

```diff
@@ -498,11 +496,9 @@ public void setAccessPolicyIds() {
     public void setAccessPolicyAC(OffsetDateTime modified, OffsetDateTime unmodified, String leaseID) {
         Mono<Response<Void>> response = setupContainerLeaseConditionAsync(ccAsync, leaseID)
             .flatMap(r -> {
-                if ("null".equals(r)) {
-                    r = null;
-                }
+                List<String> list = convertNulls(r, null);
                 BlobRequestConditions cac = new BlobRequestConditions()
-                    .setLeaseId(r)
+                    .setLeaseId(list.get(0))
                     .setIfModifiedSince(modified)
                     .setIfUnmodifiedSince(unmodified);
```

Comment:
And for the cases where there is only one condition that we are parameterizing, I think it would help with the readability if we were to introduce another helper method `convertNull` that will return a string instead of a list of strings to help us avoid having to do the additional hop of grabbing from index `0` of the list. From there we should be able to condense all of these test cases to something like this:
```java
        Mono<Response<Void>> response = setupContainerLeaseConditionAsync(ccAsync, leaseID)
            .flatMap(condition -> {
                BlobRequestConditions cac = new BlobRequestConditions()
                    .setLeaseId(convertNull(condition))
                    .setIfModifiedSince(modified)
                    .setIfUnmodifiedSince(unmodified);

                return ccAsync.setAccessPolicyWithResponse(null, null, cac);
            });

        assertAsyncResponseStatusCode(response, 200);
```
where we:
1. Change `r` to be something a bit more self documenting like `condition`
2. Do the null conversion as part of the builder to make it less pronounced in the test body